gen/drupal_elixir

```diff
@@ -147,19 +154,20 @@ sub processUsers {
 			# change from SUSPENDED to EXPIRED
 			$userStruc->{$uid}->{$u_status} = $status;
 		}
-	} else {
-		$userStruc->{$uid}->{$u_login} = $login;
-		$userStruc->{$uid}->{$u_status} = $status;
-		$userStruc->{$uid}->{$u_email} = $email;
-		#duplicity of two different users in email is not allowed
-		my $lowerCaseEmail = lc($email);
-		if($usedEmailsStruc->{$lowerCaseEmail}) {
-			$errorInDuplicity .= "Users with logins '$login' and '" . $usedEmailsStruc->{$lowerCaseEmail} . "' have same email '$email'\n";
+	} else{
+    my $lowerCaseEmail = lc($email);
+		#duplicity found, create record of duplicity accounts
+		if(exists $lowerCaseEmailStruc->{$lowerCaseEmail}) {
+      $duplicityUserStruc->{$uid}->{$u_login} = $login;
+			$duplicityUserStruc->{$uid}->{$u_email} = $email;
```

Comment:
So, the duplicates are not pushed to the normal file. Regarding the managed service - what if duplicity happens later, when first account already exist. Do we care? Is such account removed/disabled?

Reply:
We are just sending file with actual situation without duplicities. There is possibility when one account will be removed and different one will be added. I don't think we can do anything about it.

Reply:
Ok, I was just thinking aloud. If service didn't specify its preferences for solution, this should be ok.